Dockerfiles/os-nginx-frontend/nginx.conf

```diff
@@ -17,7 +16,8 @@ http {
                       '$status $body_bytes_sent "$http_referer" '
                       '"$http_user_agent" "$http_x_forwarded_for"';
 
-    access_log  /var/log/nginx/access.log  main;
+    access_log /proc/self/fd/1;
+    error_log /proc/self/fd/1 info;
```

Comment:
Nice!! This makes the logs go to the docker stdout?

Reply:
Yep, we did it for our openspending.org Nginx instance as part of the DataDog work: https://hackmd.io/4gLJxSJ3TwaDvYUy-LR66g